### PR TITLE
ElasticNet_StandardScaler_PolynomialFeatures.ipynb added

### PR DESCRIPTION
**Pull request by JayShimpi22**

### CODE REVIEW

**Comment by Thilakraj1998** [comment]
the approach has issue
you have performed StandardScaler twice
first on train and test set

then again inside the pipeline code



ss = StandardScaler()
X_train = ss.fit_transform(X_train)
X_test = ss.transform(X_test)

Remove above mentioned lines of code

**Comment by JayShimpi22** [comment]
sorry, i will rectify it
